cartographer/sensor/compressed_point_cloud.cc

```diff
@@ -178,5 +178,17 @@ proto::CompressedPointCloud CompressedPointCloud::ToProto() const {
   return result;
 }
 
+// Proto to CompressedPointCloud
+CompressedPointCloud ToCompressedPointCloud( const proto::CompressedPointCloud&
+proto) {
+  CompressedPointCloud compressed_point_colud;
+  const size_t num_points = proto.point_data_size();
```

Comment:
Tests added. I went ahead and decompressed the points to make sure what went in is what came out. Does that seem acceptable?